yarn_api_client/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-__version__ = '0.3.3'
+__version__ = '0.3.4.dev0'
```

Comment:
Thank you for adding the `dev` suffix - it was difficult to determine a dev build from a release previously.